package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/nat",
-  "version": "3.0.2-dev.0",
+  "version": "4.0.0-dev.0",
```

Comment:
Was this done manually? If so, let's not change the version manually and instead use the release instructions in CONTRIBUTING.md. Those release instructions have been altered for `yarn` rather than `npm` in https://github.com/Agoric/nat/pull/144 but may need additional fixes.